vaadin-tabs-flow-parent/vaadin-tabs-flow/src/main/java/com/vaadin/flow/component/tabs/TabSheet.java

```diff
@@ -265,6 +265,45 @@ public int getIndexOf(Tab tab) {
         return tabs.indexOf(tab);
     }
 
+    /**
+     * Returns the {@link Tab} associated with the given component.
+     *
+     * @param component
+     *            the component to look up, can not be <code>null</code>
+     * @return The tab instance associated with the given component, or
+     *         <code>null</code> if the {@link TabSheet} does not contain the
+     *         component.
+     */
+    public Tab getTab(Component component) {
```

Comment:
```suggestion
    public Tab getTab(Component content) {
```
Let's align the parameter name with existing methods, like `add` and `remove`, and also update the JavaDocs to refer to "content".

Reply:
Done